examples/index.css

```diff
@@ -1,3 +1,7 @@
+@import "~normalize.css";
```

Comment:
CSS imports don't differentiate between `./foo.css` and `foo.css` (both relative), which is why for webpack it becamse kinda standard to use the tilde prefix to signify the `node_modules` folder. Which means this directly imports the `normalize.css` package's `main` file, which happens to be correctly set to the libraries' css file :)